examples/rfm69_node1_ack.py

```diff
@@ -0,0 +1,68 @@
+# Example to send a packet periodically between addressed nodes with ACK
+# Author: Jerry Needell
+#
+import time
+import board
+import busio
+import digitalio
+import adafruit_rfm69
+
+# set the time interval (seconds) for sending packets
+transmit_interval = 5
```

Comment:
@jerryneedell with `rfm69_node1_ack.py` and `rfm69_node2_ack.py` I thought I was noticing a trend of longer streaks where I was getting several (tens) of successes in a row or several of `No Acks` in row with the opposites sprinkled in occasional but a pretty solid trend of back and forth being more or less likely to succeed. 

I started taking a peek at how it's working inside of `adafruit_rfm69.py`. I am brand new to this hardware and low level radio like this generally so I'm not sure how much to trust about my understanding and the conclusions I drew from them perhaps they may not be plausible. 

I got to wondering if the way the various delays inside of the send/retry/receive processes were shaking out was causing my devices to go through periods of being "out of sync" where node2 was not done with all of its waiting and retrying before node 1 was moving on and sending the next packet. I thought maybe increasing this `transmit_interval` at the application layer might allow some extra time and possibly alleviate it some if this was actually the case. 

I increased this timeout to `10` and let them run for a while. In my case it did seem to help out. I do still get a few `No Acks` in a row from time to time, but overall it seems like my success rate seems to have increased a good bit and I'm no longer noticing stronger trends of success and failure streaks back and forth.

Reply:
It will definitely be better with fewer transmissions. I was trying to strike a balance for testing. I think you are correct that with 5 seconds between transmissions, it can get caught up in the retries. I'm happy to set this to 10 for the demo.